src/backend/utils/indexer.js

```diff
@@ -73,11 +73,10 @@ const search = async (textToSearch) => {
     body: query,
   });
   const results = hits.total.value;
-  const values = hits.hits.map(({ _id, _source, _score }) => {
+
+  const values = hits.hits.map(({ _id }) => {
```

Comment:
You don't need the inner `return` on your arrow function, or a statement body.  Do:

```js
const values = hits.hits.map(({ _id }) => ({ id: _id }));
```

Also, I would do this all inline on 83-86:

```js
return {
  results: hits.total.value,
  values: hits.hits.map(({ _id }) => ({ id: _id })),
};
```

Reply:
I see. I just wanted to use the project's style. I will update it without return.
For test, to be honest, I need to study more~ let me take time~ Thank you.

Reply:
The https://github.com/Seneca-CDOT/telescope/blob/master/src/backend/utils/__mocks__/indexer.js will use what you initially wrote with the `.sort().map()` since we're mocking, it doesn't automatically sort by itself. Let me know if you have more questions on this part.

Also  line 183 + 184 (I think) of `feed.test.js` will also have to be removed, you'll run into the issue when you run npm test.. The tests fail at those two locations because we're not returning text anymore.

Reply:
@c3ho hm actually, I have no experience about the test code. Mocking means can I just set the data manually? 
```
const db = {
  results: 0,
  values: [{_index: 'posts', _type: 'post', _id: '68063f7971, _score: 4.2404203, _source: {}},
    {_index: 'posts', _type: 'post', _id: '68063f79xx, _score: 4.24042xx, _source: {}}, // xx is different number
    {_index: 'posts', _type: 'post', _id: '68063f79xx, _score: 4.24042xx, _source: {}},
    {_index: 'posts', _type: 'post', _id: '68063f79xx, _score: 4.24042xx, _source: {}}
  ],
};
```

Reply:
What we mean by mocking is, we want to copy what the real ElasticSearch does. The one in the mock doesn't do any sorting.

This is where you'll have to use what you wrote previously, to sort and only return the ID

Reply:
@c3ho hm then we just believe about the options _source for ES search? Confused.
Just want to make clear. 
Can I use the my data like below?
```
const db = {
  results: 5,
  values: [{_index: 'posts', _type: 'post', _id: '68063f7971, _score: 4.2404203, _source: {}},
    {_index: 'posts', _type: 'post', _id: '68063f79xx, _score: 4.24042xx, _source: {}},
    {_index: 'posts', _type: 'post', _id: '68063f79xx, _score: 4.24042xx, _source: {}},
    {_index: 'posts', _type: 'post', _id: '68063f79xx, _score: 4.24042xx, _source: {}},
    {_index: 'posts', _type: 'post', _id: '68063f79xx, _score: 4.24042xx, _source: {}},
    {_index: 'posts', _type: 'post', _id: '68063f79xx, _score: 4.24042xx, _source: {}}
  ],
};
```

Reply:
You can't input test data like that haha, you'll have to use the `indexPost` function

Reply:
@c3ho I see~ I will take a look at the indexPost.

Reply:
If you want, add the following line
`console.log(elasticPosts.results)` under line 182 and run `npm test feed.test.js` to see what the output is of your mocked `search` function.